README.rst

```diff
@@ -35,7 +35,13 @@ Basic Workflow
 Install
 -------
 
-First you need to install docfx-yaml:
+To use this forked version, install GCP docfx-yaml:
+
+.. code:: bash
+
+    pip install gcp-sphinx-docfx-yaml
+
+To use the default original, install original docfx-yaml:
```

Comment:
I think we can delete this and the command below.

Reply:
did you mean just the second part of the original snippet? or the pip install for gcp-sphinx-docfx-yaml as well?

Reply:
We don't need to say how to install the original. How to install this one is good to keep.

Reply:
Gotcha. Just got confused by the range of snippet shown on the front page D: